bridges/AppleAppStoreBridge.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+class AppleAppStoreBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'captn3m0';
+	const NAME = 'Apple App Store';
+	const URI = 'https://apps.apple.com/';
+	const CACHE_TIMEOUT = 3600; // 1h
+	const DESCRIPTION = 'Returns version updates for a specific application';
+
+	const PARAMETERS = array(array(
+		'id' => array(
+			'name' 			=> 'Application ID',
```

Comment:
Please generally don't align arrays. It just looks wrong depending on editor settings.